VIMVideoPlayer/VIMVideoPlayer.m

```diff
@@ -424,6 +426,8 @@ - (void)resetPlayerItemIfNecessary
 
 - (void)preparePlayerItem:(AVPlayerItem *)playerItem
 {
+    self.item = playerItem;
```

Comment:
would it make any sense to check if `self.item != nil` here and behave appropriately if so?